alvr/common/src/average.rs

```diff
@@ -38,3 +39,40 @@ impl SlidingWindowAverage<Duration> {
         self.history_buffer.iter().sum::<Duration>() / self.history_buffer.len() as u32
     }
 }
+
+impl SlidingWindowAverage<Vec2> {
+    pub fn get_average(&self) -> Vec2 {
+        self.history_buffer.iter().sum::<Vec2>() / self.history_buffer.len() as f32
+    }
+}
```

Comment:
I noticed just now that there's a manual impl for all different types, and why not just force `T` to implement `std::iter::Sum`? Because without that the struct is wholly useless anyway. Or if you don't wanna restrict it that much (you could honestly just impl the trait for that non-fitting type) is there no way to impl something for all `T where Trait`?

Reply:
The issue I found was casting part of the items count. And I can't just add a second generic parameter, it won't accept it with the "as" operator